media/js/mozorg/contribute/contribute-2015.js

```diff
@@ -149,10 +144,10 @@
         if ($.inArray(categoryarea, areas) !== -1) {
             if ($areascontainer.is(':hidden')) {
```

Comment:
Following the standard naming convention we do else where, should these not perhaps be `categoryArea` and `$areasContainer`?